content/en/news/release3.4.md

```
@@ -1,56 +1,74 @@
-  - TOPP tool FeatureFinderCentroided is 28-44% faster (internally GaussTraceFitter is 4-7x faster) (#7950)
+  - TOPP tool FeatureFinderCentroided TOPP tool is 28-44% faster (internally GaussTraceFitter is 4-7x faster) (#7950)
   - mzML parsing is 6% faster (#7850)
   
 Dependencies:
-- Switch to Qt 6 (>= 6.5)  (#7525)
+- switch to Qt 6 (>= 6.5)  (#7525)
 
 OpenSwath:
-- Changes to -out_features (# 7793)
+- changes to -out_features (# 7793)
 	- Deprecate .tsv feature output
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 1

(MD007, ul-indent)